airbyte-config/init/src/main/resources/seed/source_definitions.yaml

```diff
@@ -647,7 +647,7 @@
 - name: MySQL
   sourceDefinitionId: 435bb9a5-7887-4809-aa58-28c27df0d7ad
   dockerRepository: airbyte/source-mysql
-  dockerImageTag: 0.6.5
+  dockerImageTag: 0.6.6
```

Comment:
AFAIK, you should not be manually changing versions in seed files unless publish command fails. Normal flow is to get the PR approved, run "/publish" command and then github action that responds to "publish" command will update docker Image tags after it pushes the images to Docker Hub.

Reply:
I'm reverting back all image tags. thanks